privacyidea/static/components/config/views/config.policies.html

```diff
@@ -11,7 +11,14 @@
                     ng-show="checkRight('policywrite')"
                     ng-class="{ active: $state.includes('config.policies.add')}">
                     <a ui-sref="config.policies.add"
-                       translate>Create new Policy</a></li>
+                       translate>Create new Policy</a>
+                </li>
+                <hr>
+                <li role="presentation">
+                    <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                       privacyideaVersionNumber+'/policies/index.html#'}}">
```

Comment:
Please remove the trailing `#` here

Reply:
done in a37f2d98d2d6876d137ba6a6783b086b289941c3.